src/metabase/api/database.clj

```diff
@@ -478,18 +530,21 @@
   Tables are returned in the format `[table_name \"Table\"]`;
   When Fields have a semantic_type, they are returned in the format `[field_name \"table_name base_type semantic_type\"]`
   When Fields lack a semantic_type, they are returned in the format `[field_name \"table_name base_type\"]`"
-  [id prefix substring]
+  [id prefix substring tagged-card-ids]
```

Comment:
```suggestion
  [id prefix substring referenced-card-ids]
```

I don't think we have the notion of `tagged` in our current glossary.
Instead, we're referring to the `{{#123}}` as a referenced card

See : https://github.com/metabase/metabase/blob/78388b2d7059f6c463fa6799480b64ccf1430f5b/src/metabase/driver/common/parameters.clj#L37

Reply:
Great, I was hoping there was a better name for this. Thanks, I'll change it